src/content/fr/2020/performance.md

```diff
@@ -0,0 +1,388 @@
+
+## <span lang="en">First Contentful Paint</span>
+
+<span lang="en">First Contentful Paint</span> (FCP) mesure la première fois que le navigateur a affiché un texte, une image, un canvas non blanc ou un contenu SVG. Le FCP est un bon indicateur de la vitesse perçue, car il indique le temps d’attente avant de voir les premiers signes de chargement d’un site.
+
+### FCP par matériel
+
+{{ figure_markup(
+  image="performance-fcp-desktop-distribution.png",
+  caption="Distribution de sites web présentant des performances rapides, moyennes et lentes en matière de FCP sur les ordinateurs de bureau.",
+  description="Distribution des sites web ayant des performances rapides, moyennes et lentes sur le bureau. La distribution des sites web rapides semble linéaire avec un renflement au milieu. Il y a plus d’expériences de FCP rapides et lentes par rapport à 2019, tandis que le nombre d’expériences moyennes diminue en raison des changements de catégorisation du FCP.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1953305743&format=interactive",
+  sheets_gid="2122167666",
+  sql_file="web_vitals_distribution_by_device.sql"
+  )
+}}
+
+{{ figure_markup(
+  image="performance-fcp-mobile-distribution.png",
+  caption="Distribution de sites web ayant des performances rapides, moyennes et lentes sur le mobile.",
+  description="Distribution de sites web ayant des performances rapides, moyennes et lentes sur le bureau. La distribution des sites web rapides semble linéaire, avec un moindre renflement que celui observé sur le graphique des ordinateurs de bureau. Le nombre d’expériences rapides et lentes du FCP est plus élevé qu’en 2019, tandis que le nombre d’expériences moyennes diminue en raison des changements de catégorisation du FCP.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=38297781&format=interactive",
+  sheets_gid="2122167666",
+  sql_file="web_vitals_distribution_by_device.sql"
+  )
+}}
+
+Dans les graphiques ci-dessus, les distributions de FCP sont réparties entre les ordinateurs de bureau et les téléphones portables. [Par rapport à l’année dernière](../2019/performance#fcp-by-device), on constate une diminution notable des relevés FCP moyens, tandis que le pourcentage d’expériences d’utilisation rapides et lentes a augmenté quel que soit le type d’appareil. Nous pouvons toujours observer la même tendance, à savoir que les personnes utilisant des téléphones portables connaissent plus fréquemment des FCP plus lents que celles qui utilisent des ordinateurs de bureau. Dans l’ensemble, ces personnes sont plus susceptibles d’avoir une bonne ou une mauvaise expérience, plutôt qu’une expérience moyenne.
```

Comment:
If we ever do translate the 2019 chapter then the anchor link name might change. But not much we can do abut it now.